tfjs-backend-webgpu/src/binary_op_util.ts

```diff
@@ -264,6 +318,10 @@ export function getBinaryOpString(
     case BinaryOpType.SUB:
       return SUB;
     default:
-      throw new Error(`BinaryType ${type} is not implemented!`);
+      // throw new Error(`BinaryType ${type} is not implemented!`);
```

Comment:
You may put NaN-insensitive ops in another do while, and keep the throw here.

Reply:
Will do in the final cleanup, but not possible now.

```javascript
do {
  switch (type) {
    case BinaryOpType.FOOBAR:
      return FOOBAR;
    default:
      continue;
  }
  return `blah blah`;  // <-- still unreachable
} while (false);
```